### ISSUE TITLE
Speaker on an event page should be clickable

### ISSUE THREAD

**Issue by stefanmedack** [stefanmedack/cccTV#14]
At the very bottom of a detailed event, you can see the list of speakers. They are clickable already, but do not lead to a page. With the help of the search API, we could offer a page showing all presentations of one speaker.

**Comment by AndreasGB** [comment]
Hi Stefan,

I am currently trying to implement this feature, but loading a conference fails for me in my local build. I think the error comes from the c3media-base library, as their unit tests for validating a conference response also fail with a `JsonDataException`.

Can you please publish your patches to the library if you have a version newer than the current master?

Also, if you want, you can follow me back on Twitter (@Andreas4742 there) to allow me to send messages to you as a faster way of communication.

I attached the error I am receiving below:
```
W/System.err: com.squareup.moshi.JsonDataException: Expected BEGIN_ARRAY but was BEGIN_OBJECT at path $.events[0].metadata.related
W/System.err:     at com.squareup.moshi.JsonUtf8Reader.beginArray(JsonUtf8Reader.java:112)
W/System.err:     at com.squareup.moshi.CollectionJsonAdapter.fromJson(CollectionJsonAdapter.java:74)
W/System.err:     at com.squareup.moshi.CollectionJsonAdapter$2.fromJson(CollectionJsonAdapter.java:53)
W/System.err:     at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:128)
W/System.err:     at com.squareup.moshi.ClassJsonAdapter$FieldBinding.read(ClassJsonAdapter.java:205)
W/System.err:     at com.squareup.moshi.ClassJsonAdapter.fromJson(ClassJsonAdapter.java:167)
W/System.err:     at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:128)
W/System.err:     at com.squareup.moshi.ClassJsonAdapter$FieldBinding.read(ClassJsonAdapter.java:205)
W/System.err:     at com.squareup.moshi.ClassJsonAdapter.fromJson(ClassJsonAdapter.java:167)
W/System.err:     at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:128)
W/System.err:     at com.squareup.moshi.CollectionJsonAdapter.fromJson(CollectionJsonAdapter.java:76)
W/System.err:     at com.squareup.moshi.CollectionJsonAdapter$2.fromJson(CollectionJsonAdapter.java:53)
W/System.err:     at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:128)
W/System.err:     at com.squareup.moshi.ClassJsonAdapter$FieldBinding.read(ClassJsonAdapter.java:205)
W/System.err:     at com.squareup.moshi.ClassJsonAdapter.fromJson(ClassJsonAdapter.java:167)
W/System.err:     at com.squareup.moshi.JsonAdapter$2.fromJson(JsonAdapter.java:128)
W/System.err:     at com.squareup.moshi.JsonAdapter.fromJson(JsonAdapter.java:35)
W/System.err:     at retrofit2.converter.moshi.MoshiResponseBodyConverter.convert(MoshiResponseBodyConverter.java:42)
W/System.err:     at retrofit2.converter.moshi.MoshiResponseBodyConverter.convert(MoshiResponseBodyConverter.java:25)
W/System.err:     at retrofit2.ServiceMethod.toResponse(ServiceMethod.java:119)
W/System.err:     at retrofit2.OkHttpCall.parseResponse(OkHttpCall.java:218)
W/System.err:     at retrofit2.OkHttpCall.execute(OkHttpCall.java:180)
W/System.err:     at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:41)
W/System.err:     at io.reactivex.Observable.subscribe(Observable.java:10903)
W/System.err:     at retrofit2.adapter.rxjava2.BodyObservable.subscribeActual(BodyObservable.java:34)
W/System.err:     at io.reactivex.Observable.subscribe(Observable.java:10903)
W/System.err:     at io.reactivex.internal.operators.observable.ObservableSingleSingle.subscribeActual(ObservableSingleSingle.java:35)
W/System.err:     at io.reactivex.Single.subscribe(Single.java:2707)
W/System.err:     at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
W/System.err:     at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:452)
W/System.err:     at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:61)
W/System.err:     at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:52)
W/System.err:     at java.util.concurrent.FutureTask.run(FutureTask.java:266)
W/System.err:     at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
W/System.err:     at java.lang.Thread.run(Thread.java:764)
```

Kind regards,

Andreas

**Comment by stefanmedack** [comment]
Yes, Sorry for the inconvenience. There have been API changes that changed related events in Metadata from an array to an object, that leads to parsing errors in the c3media-base library.

I have removed this line in my fork of c3media-base and the latest version of develop uses the fork resources now until we found a solution for this change. I also filed an issue for the c3voc team, with the hope that they will revert the API change

https://github.com/voc/voctoweb/issues/297
